api/controllers/amritkeertan.js

```diff
@@ -95,7 +95,7 @@ exports.index = async (req, res) => {
     conn = await pool.getConnection();
     let headerID = -1;
     let header = '';
-    let out = {};
+    const out = {};
```

Comment:
can things still be added to the object?!

Reply:
yes

Reply:
const only prevents reassignment. modifying objects is fine